### PR TITLE
docs: update dydx validator quickstart

### PR DESCRIPTION
**Pull request by technicallyty**
updated the quick start docs for dydx.

### CODE REVIEW

File: docs/snippets/start-connect-dydx-snippet.mdx [hunk 1]
```diff
@@ -1,49 +1,74 @@
-The dYdX blockchain requires some additional steps to start Connect.
+**The minimum required version for Connect with dYdX is `v1.0.5+`.**
 
-**The minimum required version for Connect with dYdX is `v1.0.1+`.**
-
-First, please ensure you've received your API keys for Raydium markets. If you have not received API keys, please reach out to the team in the relevant channels.
+First, please ensure you've received your **API keys** for Raydium markets. If you have not received API keys, please reach out to the team in the relevant channels.
 
 Next, place your API keys under their corresponding URLs in the following file and save it to your system. Keep the file path handy as we will pass it into a flag when running Connect.
```
Comment (aljo242) [inline docs/snippets/start-connect-dydx-snippet.mdx:5]: What file path is this referring to?

Comment (technicallyty) [reply docs/snippets/start-connect-dydx-snippet.mdx:5]: first part of the sentence: "Next, place your API keys under their corresponding URLs in the following file and save it to your system."

File: docs/snippets/start-connect-dydx-snippet.mdx [hunk 2]
```diff
@@ -1,49 +1,74 @@
...
+   "providers":{
+      "dydx_migration_api":{
+         "name":"dydx_migration_api",
+         "api":{
+            "enabled":true,
+            "timeout":20000000000,
+            "interval":10000000000,
+            "reconnectTimeout":2000000000,
+            "maxQueries":1,
+            "atomic":true,
```
Comment (Eric-Warehime) [inline docs/snippets/start-connect-dydx-snippet.mdx]: I kind of prefer the config not having all these fields unless we need to.

**Review by Eric-Warehime — APPROVED** [review]

File: docs/snippets/start-connect-dydx-snippet.mdx (hunk 2 above)
Comment (technicallyty) [reply docs/snippets/start-connect-dydx-snippet.mdx]: fixed [remove extra fields](https://github.com/skip-mev/slinky/pull/633/commits/bec7565ec6a359dc48d4525d39846509809d8f44)

**Review by Eric-Warehime — APPROVED** [review]

**Pull request merged**